cdc/kv/main_test.go

```diff
@@ -24,6 +24,7 @@ func TestMain(m *testing.M) {
 	opts := []goleak.Option{
 		goleak.IgnoreTopFunction("github.com/pingcap/tiflow/pkg/workerpool.(*worker).run"),
 		goleak.IgnoreTopFunction("sync.runtime_Semacquire"),
+		goleak.IgnoreTopFunction("github.com/pingcap/goleveldb/leveldb.(*DB).mpoolDrain"),
```

Comment:
Can we modify the test to avoid leaking goroutine?

Reply:
No, I merely followed the solution in the TiDB repository, which also had this problem.